migrations/20221122004832_assessments__statistics__add.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE assessments ADD COLUMN OUT statistics_last_updated_at timestamptz NOT NULL DEFAULT now() - interval '100 years';
```

Comment:
I elected to cache the statistics directly in the `assessments` table, to avoid the mess when we use a second caching table but it doesn't yet have a cached row for our assessment. To hopefully keep the `assessments` table organized, I used a common prefix for all the cached rows.